src/sonic-yang-models/yang-models/sonic-vlan.yang

```diff
@@ -108,7 +108,6 @@ module sonic-vlan {
 					default disable;
 				}
 
-
                 leaf loopback_action {
```

Comment:
It seems that you don't need to modify sonic-vlan.yang

Reply:
Yes, actually i have not made any changes in sonic-vlan.yang. Just fixing these typos. Will get back with all resolved soon! Thanks again.